backend/pkg/schema/config.go

```diff
@@ -17,6 +17,9 @@ type Config struct {
 
 	// TLS / Custom CA
 	TLS TLSConfig `yaml:"tls"`
+
+	// Timeouts
+	Timeout int `yaml:"timeout"`
```

Comment:
I'm wondering whether we can use `time.Duration` immediately here. Can you try that, or aren't you able to run Kowl locally (then I can try that as well). Regardless in Go there is the concept of time.Duration where you can specify durations with strings like `5s`, `1h` etc. I think we should use this (we already use that for the `server.timeouts` for instance